eva-server/dockerconf/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM tomcat:9.0.5-jre8
```

Comment:
The alpine version would help to reduce the image size and make it more suitable for production environments.